openshift/clean.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const fs = require('fs');
+
+let rawdata = fs.readFileSync('student.json');
+let student = JSON.parse(rawdata);
+
+var args = process.argv.slice(2);
```

Comment:
Is this file supposed to be in the repo? I can't find `student.json`...

Reply:
Good catch, that's some sample code that wasn't supposed to be there.